src/components/vars.sass

```diff
@@ -66,3 +69,11 @@ $top-bar-width: 960px
 // bottom bar
 //
 $bottom-bar-height: 300px
+
+.no-jitter
+  -webkit-backface-visibility: hidden
+  backface-visibility: hidden
+  -webkit-transform: translateZ(0) scale(1.0, 1.0)
+  transform: translateZ(0)
+  filter: blur(0)
+  -webkit-filter: blur(0)
```

Comment:
This class handles the jittery text that Leslie reported.  The text jitters are a side-effect of the browser trying to optimize text rendering when are animating components and we have a scaled transform.  Related reading if interested:
https://stackoverflow.com/questions/15464055/css-transition-effect-makes-image-blurry-moves-image-1px-in-chrome
https://forum.webflow.com/t/a-fix-for-when-your-elements-move-up-and-or-blur-during-animations-interactions-on-chrome/11629